### ISSUE TITLE
CommonJS style require does not work

### ISSUE THREAD

**Issue by michaelw85** [anacierdem/vscode-requirejs#19]
```
define(function(require) {
    var moduleA = require('moduleA');
});
```

Trying to use the extension on moduleA returns not found.
I think this issue is caused by a bug in the part where it gets the list of modules + path from the define statement.

**Comment by anacierdem** [comment]
This is a duplicate of #17. Moving this example over.

**Comment by anacierdem** [comment]
Why dont you just use;

    require(['moduleA'], function(moduleA ) {

    });

**Comment by michaelw85** [comment]
I think adding the dependency array in the describe makes it hard to read when the list becomes quite long. 

The app I work with uses this as a coding standard so fixing this issue will make this extension viable for me and I'm assuming I'm not the only one. 

It is a supported way to require statements so it would be good to support it.

**Comment by anacierdem** [comment]
This is handled as a commonJS style require, instead of multiple statements.

**Comment by anacierdem** [comment]
Published 0.0.33
